src/simulator/src/circuit.ts

```
@@ -292,30 +292,41 @@ export function changeCircuitName(name: string, id = globalScope.id) {
+    ox: number
+    oy: number
+    scale: number
+    stack: any[]
+    layout: {
+        width: number
+        height: number
+        title_x: number
+        title_y: number
+        titleEnabled: boolean
+    }
+    tunnelList?: {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)